src/main/kotlin/pt/up/fe/ni/website/backend/dto/auth/ChangePasswordDto.kt

```diff
@@ -1,6 +1,12 @@
 package pt.up.fe.ni.website.backend.dto.auth
 
+import jakarta.validation.constraints.Size
+import pt.up.fe.ni.website.backend.model.constants.AccountConstants as Constants
+
 data class ChangePasswordDto(
+    @field:Size(min = Constants.Password.minSize, max = Constants.Password.maxSize)
```

Comment:
Checking the old password seems redundant. It probably won't make a difference but this could block changing passwords if we ever populate the db directly (again, shouldn't happen but ***could*** happen)

Reply:
Makes sense, removed it.